README.md

```diff
@@ -1 +1,118 @@
-# ruby-capstone-catalog
\ No newline at end of file
+<img src="https://img.shields.io/badge/Ruby-CC342D?style=for-the-badge&logo=ruby&logoColor=white" alt="ruby" height="auto" />
+
+<div align="center">
+  <h1><b>Ruby Capstone - Catalog of My Things</b></h1>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Ruby Capstone - Catalog of My Things <a name="about-project"></a>
+
+In this project I created a tool to record what books are in a library and who borrows them.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Add new students or teachers**
+- **Add new books**
+- **Save records of who borrowed a given book and when**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [N/A](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Clone project
+
+```bash
+git clone https://github.com/paulinagonzalezc/school-library-ruby
+
+cd school-library-ruby
+```
+
+Run the program
+
+```bash
+ruby main.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Paulina Gonzalez**
+
+- GitHub: [@paulinagonzalezc](https://github.com/paulinagonzalezc)
+- Twitter: [@paulinagzzc](https://twitter.com/paulinagzzc)
+- LinkedIn: [Paulina Gonzalez Cantu](https://linkedin.com/in/paulina-gonzalez-cantu)
+
```

Comment:
- Include all authors
- there are more than one contributors.

Reply:
@storres20 , Can you share your info so we can add it?